REFERENCE.md

```
@@ -61,6 +61,8 @@
   - [Force](#force)
   - [Fuel consumption](#fuel-consumption)
   - [Pressure](#pressure)
+   - [Electricity](#electricity)
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 3

(MD005, list-indent)